src/api/client.ts

```diff
@@ -36,8 +34,8 @@ import {
   CursorPaginationResponse,
   ApiListener,
   Resolver,
-  ErrorCreator, 
-  MetadataConfig,
+  ErrorCreator,
+  MetadataConfig, Media, Destination, MediaSchema, DestinationPK, DestinationRequest, NewDestination,
```

Comment:
this should probably be one per line as the rest of them, looks wierd mixing it up